src/components/AutocompleteWrapper.tsx

```diff
@@ -54,16 +55,20 @@ class AutocompleteWrapper extends Component<Props, State> {
           pathLabel: suggestion.value,
           itemLabel: suggestion.value,
           apiId: suggestion.id,
-          id: v1()
+          id: v1(),
         }))
       )
       .then(data => this.setState({ data }))
       .catch(e => console.error(e));
   };
 
-  handleSelect = (chosen: Chosen) => {
+  handleSelect = (chosen: Chosen | string) => {
```

Comment:
I know it's not part of the PR, but maybe the type name `chosen` is a bit ambiguous? `selectedValue` maybe?